### ISSUE TITLE
Headers and footers

### ISSUE THREAD

**Issue by henkmollema** [NoMoreLoad/WkWrap#1]
Is it possible to pass in HTML for header and footers in the PDF?

**Comment by NoMoreLoad** [comment]
@henkmollema Yes. You can do it calling [this method](https://github.com/NoMoreLoad/WkWrap/blob/master/WkWrap.Core/IHtmlToPdfConverter.cs#L52) and provide any [wkhtmltopdf options](https://wkhtmltopdf.org/usage/wkhtmltopdf.txt) that you want in `string settings` argument.

**Comment by henkmollema** [comment]
Thanks, I will try it out. I'll guess I have to write the header/footer content to a temporary file and pass the file name to wkhtmltopdf.
